guide/14-deep-learning/address_standerdization_using_model_extension.ipynb

```diff
@@ -0,0 +1,967 @@
+{
```

Comment:
<ul><li>Use GPT-3.5 instead of GPT 3.5 everywhere</li><li>Explain/refer the dataset used</li><li>Use the complete tool name</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2112/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Updated the information </p>